test/e2e-tests/support/options.go

```diff
@@ -55,6 +57,10 @@ func setOptions() (*Options, error) {
 		return options, fmt.Errorf("env variable %v not set, but is required to run tests", ClusterNamespaceEnvKey)
 	}
 
+	if options.SelfSignedCert = os.Getenv(SelfSignedCertEnvKey); options.SelfSignedCert == "" {
```

Comment:
Since the `ssl-cert` workspace is optional in the pipeline, do we need to block on `options.SelfSignedCert` is undefined?

Reply:
For this case we don't block if it is not set, I just print a warning saying it is not set

Reply:
Ok. I was incorrect.  The output of the missing SelfSignedCert warning was visually confusing due to it being inline with a failed test execution message